### DISCUSSION TITLE
Flag to suppress "info" phrases

### DISCUSSION THREAD

**Discussion by RndyP** [ggml-org/whisper.cpp#293] [Q&A] [answered]
Maybe I'm missing the obvious, but is there a flag to suppress all the "info" phrases, like:
(coughing)
(water splashing)
(mellow music)
etc.......

**Comment by ggerganov** [comment, marked as answer]
No flag is currently available - you will have to regex them out manually

**Reply by RndyP** [reply]
Thanks ggerganov! Doing that now. Seems like just simply rejecting anything with square brackets or parentheses cleans up most of it.
